pkg/kube/client.go

```diff
@@ -157,6 +157,7 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	objPods := c.getRelationPods(infos)
```

Comment:
This should probably be rewritten so that you get the related pods inside of the `perform` block a few lines below. Right now you are fetching an update from the API for every object and then it is happening again in the perform block

Reply:
I move the get relation pod action to the perform function.

Reply:
@thomastaylor312  Hi , thomastaylor312. I move the getRelationPods's code to the function were called it. Please help me to recheck that. so many thanks.